### ISSUE TITLE
High VL value on FastBreak

### ISSUE THREAD

**Issue by qeterme** [NoCheatPlus/Issues#357]
#### Short description of the issue.
Players get high VL values on FastBreak

#### Complete output of the _ncp version_ command (please prefer text here).
---- Version information ----
#### Server ####
git-Spigot-72c2605-709783c ~MC: 1.10.2~
  detected: 1.10.2
#### NoCheatPlus ####
Plugin: 3.15.0-SNAPSHOT-sMD5NET-b1039
MCAccess: 1.10 / Spigot-CB-1.10_R1
Features:
  blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9 | BlocksMC1_10
  checks: FastConsume | Gutenberg | HotFixFallingBlockPortalEnter | AttackFrequency | FlyingFrequency | KeepAliveFrequency
  defaults: pvpKnockBackVelocity
  packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance
Hooks: AllViolations~NCP~ 1.0
#### Related Plugins ####
ProtocolLib v4.1.0-SNAPSHOT-b308

#### How to reproduce the issue (is it random / always happens / side conditions)?
We don't know how to reproduce. We think it is random, and when we asked the players they said they didn't do anything just breaking the blocks with hand or with blocks.

#### Have you made any (noteworthy) changes to the default configuration of NCP?
Yes, I've translated and I've added so much actions.

#### Do you run any other special plugins that affect game mechanics next to NC+ (adding skills, machines, adding/changing blocks, other anti cheating)?
Nope.

#### Provide additional information, such as error/debug/violation logs. (For reproducible issues with moving consider: https://github.com/NoCheatPlus/Docs/wiki/Debugging#on-the-fly-debug-output-for-individual-players) Use GitHub gists or http://pastebin.com/ or similar for huge logs, or attach a (zip-) file here. For image uploads, please prefer a site that allows viewing without JavaScript, you can upload file attachments for issues right here.
Violation log (with only fastbreaks): https://gist.github.com/qeterme/cfd1594c60b251752124adb309fbd2f1
Our config: https://gist.github.com/qeterme/903afa25d0bf59680808156183d4a3e0

**Comment by Xayanix** [comment]
I can confirm, i had to disable fastbreak checks because of many false positives. Also when you are punching yourself in water you get teleported back.

**Comment by asofold** [comment]
Concerning the fastbreak issue, i can't tell what happens there - is it players with very laggy connections or extreme latency? You could set the parameter strict to false , so they can always break the first block instantly (!), and then the time between block breaking is measured instead between interaction/damage and block breaking.

@Xayanix The water/teleportation issue might be another thing. How to punch yourself in water?

**Comment by Xayanix** [comment]
@asofold 
Just get bow with Punch enchant and shot with lowest possible power to your feet.

**Comment by Xayanix** [comment]
Here is my issue: https://github.com/NoCheatPlus/Issues/issues/362

**Comment by asofold** [comment]
Sorry, bedrock can't be broken normally (id 4). This should be the reason for the violations - the violation level still should be capped though. Could kick directly for unbreakables (...).

Has a plugin perhaps sent other blocks to the client than there were?

I'll close as superseded by other issues, i'll open another related one (INDEV).
